src/main/target/ALIENFLIGHTF4/config.c

```diff
@@ -27,6 +27,7 @@
 #include "config/feature.h"
 
 #include "drivers/pwm_esc_detect.h"
+#include "drivers/compass/compass.h"
```

Comment:
./src/main/target/ALIENFLIGHTF4/config.c:56:5: warning: implicit declaration of function 'compassConfigMutable' [-Wimplicit-function-declaration]
     compassConfigMutable()->mag_hardware = MAG_NONE;            // disabled by default
     ^~~~~~~~~~~~~~~~~~~~